gromacs/templates/gromacswrapper.cfg

```diff
@@ -21,19 +21,7 @@ release = 5.1.1
 # tools contains the file names of all Gromacs tools for which classes are generated.
 # Editing this list has only an effect when the package is reloaded.
 # (Generated with 'ls [^Gac]*' from the Gromacs bin dir)
-tools = 
-	gmx:cluster     gmx:dyndom      gmx:mdmat       gmx:principal  gmx:select   gmx:wham        gmx:mdrun               gmx:convert-tpr
-	gmx:do_dssp     gmx:clustsize   gmx:enemat      gmx:protonate  gmx:gangle   gmx:wheel       gmx:mdrun_d             gmx:trjcat
-	gmx:editconf    gmx:confrms     gmx:energy      gmx:mindist    gmx:rama     gmx:sham        gmx:x2top               gmx:trjconv
-	gmx:eneconv     gmx:covar       gmx:filter      gmx:morph      gmx:rdf      gmx:sigeps      gmx:solvate             gmx:pdb2gmx
-	gmx:anadock     gmx:current     gmx:gyrate      gmx:msd        gmx:sorient  gmx:genconf     gmx:insert-molecules
-	gmx:anaeig      gmx:density     gmx:h2order     gmx:nmeig      gmx:rms      gmx:spatial     gmx:genion      
-	gmx:analyze     gmx:densmap     gmx:hbond       gmx:nmens      gmx:rmsdist  gmx:spol        gmx:genrestr    
-	gmx:angle       gmx:dielectric  gmx:helix       gmx:nmtraj     gmx:rmsf     gmx:tcaf        gmx:check       
-	gmx:bar         gmx:helixorient gmx:order       gmx:rotacf     gmx:traj     gmx:dump        gmx:trjorder
-	gmx:dipoles     gmx:kinetics    gmx:pme_error   gmx:rotmat     gmx:tune_pme gmx:grompp      gmx:mk_angndx
-	gmx:bundle      gmx:disre       gmx:lie         gmx:polystat   gmx:saltbr   gmx:vanhove     gmx:make_edi    
-	gmx:chi         gmx:distance    gmx:potential   gmx:sas        gmx:velacc   gmx:make_ndx    gmx:xpm2ps
+tools = gmx gmx_d
```

Comment:
This breaks the existing file format. Can we add a check somewhere that raises an error (`ValueError`?) if we detect entries with `gmx[^:]*:.*` and then say that they need to change their input file, see URL to new configuration docs?